mixins/BaseMixin.ts

```diff
@@ -58,17 +58,17 @@ export default class BaseMixin extends Vue {
   }
 
   get rootName(): string {
-    if (this.isConsumer) {
+    if (this.consumer && this.isConsumer) {
```

Comment:
You can do this in a getter and avoid condition repetition.

Reply:
added getter